pkg/yamltemplate/filetests/ytt-library/overlay/array-output-position.yml

```diff
@@ -0,0 +1,74 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+
+#@ def/end test1_left():
+---
+clients:
+  - needsSecret: true
+
+#@ def/end test1_right():
+#@overlay/match by=overlay.all
+---
+clients:
+  #@overlay/match by=overlay.subset({"needsSecret": True})
+  - needsSecret: false
+
+#@ def/end test2_left():
+---
+clients:
+  - needsSecret: true
+
+#@ def/end test2_right():
+#@overlay/match by=overlay.all
+---
+clients:
+  #@overlay/match by=overlay.subset({"needsSecret": True})
+  #@overlay/replace
+  - secret: foo
+
+#@ def/end test3_left():
+---
+clients:
+  - needsSecret: true
+
+#@ def return_something():
+#@   return "something"
+#@ end
+
+#@ def/end test3_right():
+#@overlay/match by=overlay.all
+---
+clients:
+  #@overlay/match by=overlay.subset({"needsSecret": True})
+  - #@ return_something()
+
+---
+test1
+--- #@ template.replace(overlay.apply(test1_left(), test1_right()))
+---
+test2
+--- #@ template.replace(overlay.apply(test2_left(), test2_right()))
+---
+test3
+--- #@ template.replace(overlay.apply(test3_left(), test3_right()))
+---
+
++++
+
+OUTPUT POSITION:          stdin:45 | [doc]
```

Comment:
should we add a newline after OUTPUT POSITION? seems like it would read way better with it.

Reply:
Good idea 💥